package.json

```diff
@@ -53,6 +53,7 @@
     "@reduxjs/toolkit": "^1.8.5",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "12.1.5",
+    "@testing-library/user-event": "12.1.5",
```

Comment:
I think the newest version of user-event is on 14. 
The set up is a little different if you upgrade: https://testing-library.com/docs/user-event/setup

Reply:
Yeah, I was trying to keep it on the same version of react testing-library. 

I know we downgraded testing-library since we're not on react 18 yet.

Reply:
Looks like I was able to upgrade it here
https://github.com/cfpb/ccdb5-ui/pull/444/commits/994d6783b9e323e840d2f43efbb78d9742ca744b

There was some minor change but took some time to figure out the syntax.  We can update this in explorer too.